Documentation/QuickInstall/TheInstallTool/Index.rst

```diff
@@ -76,6 +76,13 @@ The Admin Tool
       Proceed with :ref:`install-introduction-package`
 
 
+#. In the last step you verify the image handling settings under the configuration presets 
```

Comment:
That's not the last step for setup however. It's a check which can be done independently from the setup.

Reply:
That's true but still this is not part of the installation setup process. We should mention this separately after the official steps.